docs/DOCKER.md

```diff
@@ -78,27 +78,27 @@ persist through container restarts.
 You can also optionally manually specify a local folder to be used as a volume:
 
 ```shell
-⛰  docker create --name=mylndcontainer -v /media/lnd-docker/:/root/.lnd myrepository/lnd [command-line options]
+docker create --name=mylndcontainer -v /media/lnd-docker/:/root/.lnd myrepository/lnd [command-line options]
 ```
 
 ## Example
 
 Here is an example testnet `lnd` that uses Neutrino:
 
 ```shell
-⛰  docker run --name lnd-testnet myrepository/lnd --bitcoin.active --bitcoin.testnet --bitcoin.node=neutrino --neutrino.connect=faucet.lightning.community
+docker run --name lnd-testnet myrepository/lnd --bitcoin.active --bitcoin.testnet --bitcoin.node=neutrino --neutrino.connect=faucet.lightning.community
 ```
 
 Create a wallet (and write down the seed):
 
 ```shell
-⛰  docker exec -it lnd-testnet lncli create
+docker exec -it lnd-testnet lncli create
 ```
 
 Confirm `lnd` has begun to synchronize:
 
 ```shell
-⛰  docker logs lnd-testnet
+docker logs lnd-testnet
```

Comment:
I think for commands that are followed by their output, it would be helpful to mark the command with a shell prefix, maybe just `$` here.

Reply:
Makes sense, so like `$ docker logs lnd-testnet` 

I do worry does this introduce a bit of style inconsistency though? e.g. https://github.com/lightningnetwork/lnd/blob/master/docs/INSTALL.md?plain=1#L388 

On one side, I see $ as semantically correct. On the other, the sporadic usage of $ may still confuse newbies. So another option is to just do this in the other few places we include sample output

Thinking:
1. Could replace this single instance
2. Leave as-is, as the output arguably does look like output as opposed to a command
3. Replace this instance, as well as all instances where sample output is included to prefix with $

Wdyt?

Reply:
I think as long as it's consistent in that if there's example output we place a `$` in front of the command, it should be somewhat understandable. So what I had in mind was option 3.

Reply:
[[@guggero wanted to run this by you one more time before I make the changes, because I was having some second thoughts as I stumbled upon each example; the original intention of the PR was to make copy and pasting easier and I have a concern that this semantically correct change detracts from the usability of the documentation.

I think it is unlikely for a user to c+p one of the outputs because they will see the output before they execute the command. Or worst case, they run all the lines at once, and they still receive the expected output, as well as some type of blank output from the CLI for the improper lines.

Also smaller, more pedantic, thing I don't think it will be possible to verify-script the change cleanly

Reply:
You're right, this doesn't look very consistent. But without the `$` sign, in some examples it's very easy to overlook that there's a command at the beginning. See here for example: https://github.com/lightningnetwork/lnd/blob/8db9ef8a1beca7ca21ba1010bd4698d6b59c7b2c/docs/configuring_tor.md
That `tor` command at the start of the first block just looks lost.

That's why I would even go ahead and just replace the mountain emoji with a `$` sign everywhere. I think that might solve the original copy/paste issue since the dollar will just resolve to an empty string in most shells so you're able to copy/paste multiple lines.

Reply:
>since the dollar will just resolve to an empty string 

Genius! And it can be verified via script. Will do shortly! Thank you sir

Reply:
Hmm... Maybe I should've tested this first and not just speculate on it. Looks like at least `bash` doesn't like it... `$: command not found`

Reply:
Ah yeah, I tried this on zsh as well, and by default, it gives command not found.

Otherwise, would've liked the solution as well. 

I definitely see your concern, but I think when even looking at the tor example from a usability standpoint, the context is quite helpful:
>you'll then want to start the Tor daemon:

And we do have some instances where lnd shows up (almost as) naked: https://github.com/lightningnetwork/lnd/blob/8db9ef8a1beca7ca21ba1010bd4698d6b59c7b2c/docs/watchtower.md?plain=1#L63 

I think it comes down to a tradeoff between semantic correctness (we do the $ swap) perhaps and usability (merge as-is)... and would have to defer what the usual direction is here?

Reply:
I gave this some more thought. I think the main concern of the issue author was to make it easy to copy/paste multiple lines of the installation instructions. My main concern is being able to easily understand our docs (we do all the support if a user _doesn't_ understand the docs). So I feel like a clearly defined shell command should be visible as such.

How about the following compromise:
 - We do one commit that replaces the mountain emoji with the standard `$` symbol (can be scripted).
 - We add another commit that removes the `$` from a few key places that have the following properties:
    - It's a block of commands only (no example output)
    - They are commands that will be copied very often and don't necessarily need to be understood in any detail (i.e. an installation instruction and not a beginner tutorial). Because my personal opinion is still that we shouldn't train users to blindly copy/paste shell commands from the web. And especially for tutorials/HOWTOs it's important the users understand what each command does so forcing them to copy/paste each line individually isn't necessarily a bad thing.

What do you think? It looks like mostly `docs/INSTALL.md` and `docs/DOCKER.md` would qualify for the second commit.

Reply:
Yep, I think that sounds like a happy medium 👍 and practically strikes a good balance. Agreed, I was concerned this goes a little too far in enabling blind c/p behavior. This allows best of both worlds. Will update soon! Thanks